server/lib/get-contract-by-id.js

```diff
@@ -102,29 +102,30 @@ module.exports = exports = async (contractId, locals = {}) => {
 			return decorateContract(contract_data, locals.hasGraphicSyndication);
 		}
 	}
-	let contract = await getSalesforceContractByID(contractId);
-	if (contract.success === true) {
-		contract = reformatSalesforceContract(contract);
-		contract.last_updated = new Date();
-		if (contract.orders) {
+
+	let sfContract = await getSalesforceContractByID(contractId);
+	if (sfContract.success === true) {
+		let reformattedContract = reformatSalesforceContract(sfContract);
+		reformattedContract.last_updated = new Date();
+		if (sfContract.orders) {
 			const currentTimeInMilliseconds = new Date();
-			const activeOrder = contract.orders.find(order => order.status === 'Activated' && new Date(order.startDate) <= currentTimeInMilliseconds && new Date(order.endDate) >= currentTimeInMilliseconds);
-			contract.current_start_date = new Date(activeOrder.startDate);
-			contract.current_end_date = new Date(activeOrder.endDate);
+			const activeOrder = sfContract.orders.find(order => order.status === 'Activated' && new Date(order.startDate) <= currentTimeInMilliseconds && new Date(order.endDate) >= currentTimeInMilliseconds);
+			reformattedContract.current_start_date = new Date(activeOrder.startDate);
```

Comment:
**Question:** Did you mean to set the `current_start_date` of `sfContract` here instead like the previous code was doing?

Reply:
no.. what I intended is for `sfContract` to remain as the salesforce contract without it being mutated. `reformattedContract` is the data we are preparing for the database, and then `mappedContract` is for the query to insert to the database.